common/views/pages/_document.tsx

```diff
@@ -60,20 +64,8 @@ class WecoDoc extends Document {
     return (
       <Html lang="en" className="is-keyboard">
         <Head>
-          {/* GA v4 */}
-          <script
-            async
-            src="https://www.googletagmanager.com/gtag/js?id=G-206J7SLYFC"
-          ></script>
-          <script
-            dangerouslySetInnerHTML={{
-              __html: `
-                  window.dataLayer = window.dataLayer || [];
-                  function gtag(){dataLayer.push(arguments);}
-                  gtag('js', new Date());`,
-            }}
-          />
-
+          <GoogleAnalyticsV4 />
+          <GoogleAnalyticsUA />
```

Comment:
I have a vague memory that we don't components certain elements in the `<head>` (I think `<meta>` tags because they have to be declare at the top level; I assume that's not an issue here?

Reply:
It seems to work fine; it does seem like the way the nextjs `Head` component works and what it tolerates might've changed/expanded, respectively, over the years